ui/app/styles/components/tabs.scss

```diff
@@ -44,3 +44,29 @@
     outline: none;
   }
 }
+
+.nav-tab-link {
```

Comment:
Creating/using this class allowed me to remove the weird nesting doll `<LinkTo>` situation.